x/fundraising/keeper/bid.go

```diff
@@ -254,13 +258,17 @@ func (k Keeper) ModifyBid(ctx sdk.Context, msg *types.MsgModifyBid) error {
 		return sdkerrors.Wrap(sdkerrors.ErrInvalidRequest, "bid price and coin amount must be changed")
 	}
 
+	if err := k.PayPlaceBidFee(ctx, msg.GetBidder()); err != nil {
+		return err
+	}
+
```

Comment:
IMO, `ModifyBid` doesn't have to impose `PlaceBidFee` since it does not create a new struct, rather it just modifies the existing `Bid` struct.

Reply:
Thinking of it now, i agree that it doesn't need to impose fees when a bidder wants to modify their bid. I'll remove it.